config/exercise_readme.go.tmpl

```diff
@@ -18,6 +18,19 @@ To raise a message with an exception, just write it as an argument to the except
 raise Exception("Meaningful message indicating the source of the error")
 ```
 
+## Running the tests
+
+To run the tests run the command `python -m pytest {{ .Spec.SnakeCaseName }}_test.py`
+(`py.test {{ .Spec.SnakeCaseName }}_test.py` for Python 2; `pytest <exercise>_test.py` for Python 3). Or run it with
+`-v` to be more verbose like this: `pytest {{ .Spec.SnakeCaseName }}_test.py`. Using the
```

Comment:
This line doesn't make sense. It seems to be explaining the verbose flag, but doesn't use it in the example.